src/Domain/User/Event/UserWasCreated.php

```diff
@@ -6,59 +6,24 @@
 
 use App\Domain\Shared\ValueObject\DateTime;
 use App\Domain\User\ValueObject\Auth\Credentials;
-use App\Domain\User\ValueObject\Auth\HashedPassword;
-use App\Domain\User\ValueObject\Email;
-use Assert\Assertion;
-use Broadway\Serializer\Serializable;
-use Ramsey\Uuid\Uuid;
 use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Serializer\Annotation\Groups;
```

Comment:
Having a framework specific class as a dependency in domain isn't something I see very clear

Reply:
I prefer a yaml file or xml instead